### PR TITLE
Feat/new insti cards

### PR DESCRIPTION
**Pull request by Thiemann96**
New institutions cards according to the [design document](https://www.figma.com/design/rq5IOcBWjVAl7tmvtq0GqS/Use-Cases?node-id=7644-10183&t=rHoAjipkZlYoYzfE-4). 

For some reason I can not seem to get the select right. I tried and tried but the select 2 weirds bars sticking out. Maybe one of you can help me out with that, please? 

grafik

### CODE REVIEW

**Comment by jona159** [comment]
Maybe we could add some fixed width or something like that to the cards? For me it looks like this in the preview
WidthInstiCards

**Comment by Thiemann96** [comment]
Yeah Umut told me the same. For me I can not create issuers with a description with less than 200 characters so this problem never occured to me 😅 

I will edit some issuers in the admin tool for testing and give the cards a fixed width, thanks for the suggestion

**Comment by Thiemann96** [comment]
Learning path count is fixed [here](https://github.com/mint-o-badges/badgr-server/pull/278)

File: src/app/catalog/components/issuer-catalog/issuer-catalog.component.html [hunk 1]
```diff
@@ -1,319 +1,105 @@
 <form-message></form-message>
-
-<header class="topbar">
-	<div class="l-containerxaxis topbar-x-wrap">
-		<h2 class="topbar-x-heading">
+<div class="oeb page-padding">
+	<header class="oeb-headline-container tw-relative oeb-section-sm">
+		<h2 class="tw-flex tw-flex-col tw-text-[#492E98] tw-font-black tw-text-5xl tw-font-[rubik]">
```
Comment (umut0) [inline src/app/catalog/components/issuer-catalog/issuer-catalog.component.html:4]: please use tailwind variables for the colors.

Comment (umut0) [inline src/app/catalog/components/issuer-catalog/issuer-catalog.component.html:4]: Use the predefined text sizes: eg. hlmH2, hlmP for all texts

File: src/app/catalog/components/issuer-catalog/issuer-catalog.component.html [hunk 2]
```diff
@@ -1,319 +1,105 @@
...
-									alt="{{ issuer.name }} avatar"
+						<div class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-gap-6">
+							<div class="tw-flex tw-relative tw-items-center">
+								<input
+									hlmInput
+									class="tw-bg-white tw-w-full tw-border-1 tw-border-[#492E98] md:tw-w-96 tw-border-solid tw-min-h-12 tw-rounded-lg tw-p-2 tw-font-[rubik]"
+									placeholder="{{ 'Issuer.searchInstitutions' | translate }}"
+									[(ngModel)]="searchQuery"
 								/>
+								<hlm-icon
```
Comment (umut0) [inline src/app/catalog/components/issuer-catalog/issuer-catalog.component.html:29]: maybe we should add icon option to oeb-input

File: src/app/institutions-card/institutions-card.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<div
```
Comment (umut0) [inline src/app/institutions-card/institutions-card.component.html:1]: Move these files to the right folder: common/components/ i would suggest

File: src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import type { ClassValue } from 'clsx';
 
 export const inputVariants = cva(
-	'tw-flex tw-box-border tw-rounded-[10px] tw-border tw-font-normal tw-border-input tw-bg-transparent tw-text-sm  file:tw-border-0 file:tw-text-foreground file:tw-bg-transparent file:tw-text-sm file:tw-font-medium placeholder:tw-text-muted-foreground focus-visible:tw-outline-none focus-visible:tw-ring-2 focus-visible:tw-ring-purple focus-visible:tw-ring-offset-2 disabled:tw-cursor-not-allowed disabled:tw-opacity-50',
+	'tw-flex tw-box-border tw-rounded-[10px] tw-border tw-font-normal  tw-bg-transparent tw-text-sm  file:tw-border-0 file:tw-text-foreground file:tw-bg-transparent file:tw-text-sm file:tw-font-medium placeholder:tw-text-muted-foreground focus-visible:tw-outline-none focus-visible:tw-ring-2 focus-visible:tw-ring-purple focus-visible:tw-ring-offset-2 disabled:tw-cursor-not-allowed disabled:tw-opacity-50',
```
Comment (umut0) [inline src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts:7]: this change breaks buttons on other pages. e.g. login page.

File: src/app/components/spartan/ui-button-helm/src/lib/hlm-button.directive.ts [hunk 5]
```diff
@@ -28,7 +29,7 @@ export const buttonVariants = cva(
 				sm: 'tw-py-[6px] tw-px-[20px] md:tw-rounded-[10px] tw-rounded-[7px] tw-text-[16px] tw-leading-[24px] tw-border tw-font-bold',
 				xs: 'tw-py-[4px] tw-px-[16px] md:tw-rounded-[10px] tw-rounded-[7px] tw-border',
 				xxs: 'tw-py-[3px] tw-px-[9px] md:tw-rounded-[10px] tw-rounded-[7px] tw-border',
-				icon: 'tw-h-10 tw-w-10',
+				icon: 'tw-h-12 tw-w-12 ',
```
Comment (umut0) [inline src/app/components/spartan/ui-button-helm/src/lib/hlm-button.directive.ts:32]: Did you check if this breaks other pages?

**Review by umut0 — CHANGES_REQUESTED** [review]

**Comment by umut0** [comment]
image
Make these same height please.

**Comment by umut0** [comment]
Also maybe rename to issuer-card. That is the name used so far for those kind of entities.

File: src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts (hunk 4 above)
Comment (Thiemann96) [reply src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts:7]: ok weird i was under the impression that all inputs use <oeb-input> for styling ..

File: src/app/components/spartan/ui-button-helm/src/lib/hlm-button.directive.ts (hunk 5 above)
Comment (Thiemann96) [reply src/app/components/spartan/ui-button-helm/src/lib/hlm-button.directive.ts:32]: yes. size="icon" is only used in `issuer-catalog.component.html` and nowhere else

File: src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts (hunk 4 above)
Comment (Thiemann96) [reply src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts:7]: just found out that you can prioritize classes in tw with `!tw-bg-transparent`

File: src/app/catalog/components/issuer-catalog/issuer-catalog.component.html (hunk 2 above)
Comment (Thiemann96) [reply src/app/catalog/components/issuer-catalog/issuer-catalog.component.html:29]: would be nice, i have not found any other input where an icon is needed but good to keep in mind for the future

**Review by umut0 — APPROVED** [review]

**Comment by umut0** [comment]
Ran into some conflicts with icon buttons when i tried to merge. Fixed it i think with new iconLeft option for buttons.

**Pull request merged**